src/components/Card/Card.js

```diff
@@ -1,16 +1,11 @@
 import React from "react";
 import "./Card.css";
 
-const Card = ({ name, race, gender, realm, birthday, wikiUrl, id }) => {
+const Card = ({ dialog, key, id }) => {
```

Comment:
I don't believe you need to pass 'key' in here? 🤔

Reply:
I think I need key for all my child objects... but I will experiment